data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -56,12 +56,16 @@ public ClientImpl(ConnectionStringBuilder csb, CloseableHttpClient httpClient) t
         Objects.requireNonNull(clusterUrlForParsing.getAuthority(), "clusterUri.authority");
         String auth = clusterUrlForParsing.getAuthority().toLowerCase();
         if (host == null && auth.endsWith(FEDERATED_SECURITY_SUFFIX)) {
-            csb.setClusterUrl(new URIBuilder().setScheme(clusterUrlForParsing.getScheme()).setHost(auth.substring(0, clusterUrlForParsing.getAuthority().indexOf(FEDERATED_SECURITY_SUFFIX))).toString());
+            csb
```

Comment:
59-60 is an odd formatting decision

Reply:
point is without the new line it would have been shorter.
How much control do we have here

Reply:
Ok, I chose a new style which seems to be as good as it gets.

The downside is that I had to generate it directly from eclipse, so the comments are gone.
But now I know that their interface is fairly simple to understand and edit, so it's no big deal.


You can just look at the diff of the last commit to see if the changes are what you had in mind.